src/Http/Response.php

```diff
@@ -62,8 +62,15 @@ public function getHtmlContent()
 
                 $this->htmlContent = new DOMDocument();
 
-                if (mb_detect_encoding($content) === 'UTF-8') {
+                if (mb_detect_encoding($content, 'UTF-8', true) === 'UTF-8') {
                     $content = mb_convert_encoding($content, 'HTML-ENTITIES', 'UTF-8');
+                    $content = preg_replace(
+                        '/<head[^>]*>/',
+                        '<head><META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">',
+                        $content
+                    );
+                } elseif (mb_detect_encoding($content, 'ISO-8859-1', true) === 'ISO-8859-1') {
+                    $content = iconv('iso-8859-1', 'utf-8', $content);
```

Comment:
Why are you using `iconv` instead `mb_convert_encoding` like in the case above?

Reply:
I had no particular reason, but I changed and tested and it worked. Tks!